pkg/provider/gcp.go

```diff
@@ -109,7 +145,9 @@ func (gcp *GCPProvider) CreateBackend(prefix string, ctx map[string]interface{})
 	}
 
 	ctx["Project"] = gcp.Project()
-	ctx["Location"] = gcp.Region()
+	// Location is here for backwards compatibility
+	ctx["Location"] = getLocation()
```

Comment:
we probably don't want to shell out to gcloud every time this is run.  Ideally we'd also put this in the context once and fetch it from there.

Reply:
Actually we definitely don't want to be shelling to gcloud here, because it'll break in the console

Reply:
Yeah I wasn't sure about that either. I'll put it in the context in the morning, but for testing that change shouldn't make much difference.